pyprep/utils.py

```diff
@@ -232,7 +232,7 @@ def _eeglab_fir_filter(data, filt):
         )
 
     # Finish filtering data, using padding at end to calculate final values
-    end_pad_idx = np.zeros(pad_len, dtype=np.uint8) + (n_samples - 1)
+    end_pad_idx = np.zeros(pad_len, dtype=np.uint) + (n_samples - 1)
```

Comment:
luckily, changing from uint8 to uint does not change anything ... probably because this was done already in the background before numpy 2.0